client/auth/login.jsx

```diff
@@ -135,7 +162,7 @@ module.exports = React.createClass( {
 							<FormTextInput
 								name="login"
 								ref="login"
-								disabled={ requires2fa || inProgress }
+								disabled={ Boolean( requires2fa ) || inProgress }
```

Comment:
thought I don't think we have a policy on this, I'm not a big fan of using `Boolean()` and favor using `!!` both because I feel that it's more idiomatic in JavaScript but also because it butts up against a big JS wut

```js
Boolean( true ) || 'busted' // true
Boolean( false ) || 'busted' // 'busted'
(new Boolean( false )) || 'busted' // [Boolean: false]
```